### ISSUE TITLE
Add french quote marks on Ù key

### ISSUE THREAD

**Issue by just1602** [ergosteur/cf-keylayout#7]
I've tried to bind french quote marks but I didn't find the Ù key code to edit. Can you add it?

Thanks!

Justin : )

**Comment by joallard** [comment]
Umm, it's working already on [line 813](https://github.com/ergosteur/cf-keylayout/blob/b3883378f967f82fb4b20acbae19b9acecadf0b4/cf.keylayout#L813), what's wrong?

Sequence: `` ` ``, `Shift` + `U`

**Comment by just1602** [comment]
What I wanted to do was bind « on the `ù` key and » on the `Shift + u` key.

Sorry for the bad explanation. :/

Justin : )

**Comment by joallard** [comment]
> What I wanted to do was bind `«` on the `ù` key, and `»` on the `Shift + ù` combo.
![](http://i.stack.imgur.com/zdbJZ.jpg)

I don't have that keyboard (my English one doesn't have a key between Z and Left-Shift), but why not implement that on `Alt+Z` and `Alt+X` instead?

**Comment by just1602** [comment]
`Alt+z` and `Alt+x` would be perfect!

I'll check it out!

Thanks!

Justin : )

**Comment by joallard** [comment]
I guess leaving this open until we implement that in would make sense, yes?

**Comment by just1602** [comment]
Yes, I closed it because I didn't know if you planned to implement it. I personnaly tried yesteday and I'm doing something wrong, because I'm not able to make it work... :/

**Comment by joallard** [comment]
Okay, you can PR it and I'll take a look!

**Comment by stef-levesque** [comment]
This works for me.
https://github.com/stef-levesque/cf-keylayout/commit/aeb92e527fc29f73f63aaa42e076802b0809e6e6

**Comment by just1602** [comment]
I exactly tries this, but it didn't work, but when I loaded and installed stef-levesque version everything work.

Thanks!

Justin : )

**Comment by stef-levesque** [comment]
I notice that you need to change the <keyboard> name attribute for OS X to really update the Input source. I usually just bump the minor version number.
